core/src/main/java/com/novoda/noplayer/exoplayer/NoPlayerExoPlayerCreator.java

```diff
@@ -51,6 +52,8 @@ ExoPlayerTwoImpl create(Context context, DrmSessionCreator drmSessionCreator, bo
 
             DefaultTrackSelector trackSelector = new DefaultTrackSelector();
 
+            MediaCodecSelector mediaCodecSelector = downgradeSecureDecoder ? SecurityDowngradingCodecSelector.newInstance() : MediaCodecSelector.DEFAULT;
```

Comment:
`boolean downgradeSecureDecoder` is transformed into the associated `MediaCodecSelector`.